xpk.py

```diff
@@ -4228,7 +4228,7 @@ def cluster_create(args) -> int:
   xpk_exit(0)
 
 
-def cluster_delete(args) -> int:
+def cluster_delete(args):
```

Comment:
nit`  -> None`

Reply:
Addressed and took care of the other high level functions also.